bip-0090.mediawiki

```diff
@@ -0,0 +1,119 @@
+
+==Motivation==
+
+Most multi-signature transactions are addressed to P2SH (pay-to-script-hash) addresses, as defined in BIP-0016.
+
+Multi-signature redeem scripts do not require a particular ordering or encoding for public keys.  This means that for a given set of keys and number of required signatures, there are as many as 2(n!) possible standard redeem scripts, each with its separate P2SH address.  Adhering to an ordering and key encoding would ensure that a multi-signature “account” (set of public keys and required signature count) has a canonical P2SH address.
+
+By adopting a sorting and encoding standard, compliant wallets will always produce the same P2SH address for the same given set of keys and required signature count, making it easier to recognize transactions involving that multi-signature account.  This is particularly attractive for multisignature hierarchical-deterministic wallets, as less state is required to setup multi-signature accounts:  only the number of required signatures and master public keys of participants need to be shared, and all wallets will generate the same addresses.
+
+While most web wallets do not presently facilitate the setup of multisignature accounts with users of a different service, conventions which ensure cross-compatibility should make it easier to achieve this.
+
+Many wallet as a service providers use a 2of3 multi-signature schema where the user stores 1 of the keys (offline) as backup while using the other key for daily use and letting the service cosign his transactions.  
+This standard will help in enabling a party other than the service provider to recover the wallet without any help from the service provider.
+
+==Specification==
+
+For a set of public keys, ensure that they have been received in compressed form:
+    022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da
+    03e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e9 
+    021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc18
+ 
+Sort them lexicographically according to their binary representation: 
+    021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc18
+    022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da
+    03e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e9
+
+..before using the resulting list of keys in a standard multisig redeem script:  
+    OP_2 021f2f6e1e50cb6a953935c3601284925decd3fd21bc445712576873fb8c6ebc18 022df8750480ad5b26950b25c7ba79d3e37d75f640f8e5d9bcd5b150a0f85014da 03e3818b65bcc73a7d64064106a859cc1a5a728c4345ff0b641209fba0d90de6e9 OP_3 OP_CHECKSIG
+
+Hash the redeem script according to BIP-0016 to get the P2SH address.
+    3Q4sF6tv9wsdqu2NtARzNCpQgwifm2rAba
```

Comment:
Thanks, seems to have been the same for the first set of public keys too. Fixed now